nginx_proxy/config.yaml

```
@@ -1,4 +1,4 @@
-version: 3.0.2
+version: 3.1.0
```

```suggestion
version: 3.1.0
hassio_api: true
```
Needed to use the supervisor API